core/templates/dev/head/css/oppia.css

```diff
@@ -4045,13 +4045,13 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-donation-card {
-  margin: auto;
+  margin: 0px auto 30px auto;
```

Comment:
Use just 0, not 0px.

(Though I thought you mentioned that setting a margin-bottom didn't work?)